src/mono/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.PortableThreadPool.Mono.cs

```diff
@@ -13,6 +13,47 @@ namespace System.Diagnostics.Tracing
     // To look at CoreCLR implementation of these events, refer to NativeRuntimeEventSource.PortableThreadPool.CoreCLR.cs.
```

Comment:
Could you keep this file in the shared partition with the CoreCLR condition? It is useful for CoreRT too.

Reply:
(I have done very similar change in CoreRT to fix build breaks - https://github.com/dotnet/runtimelab/pull/707/commits/4089184ac5cee5146b690329d396fa3286124db1 . I just did not have a chance to push it back to dotnet/runtime.)

Reply:
OK I see.

So if we share what's in NativeRuntimeEventSource.PortableThreadPool.Mono.cs and only include it on none CoreCLR builds you will be fine in CoreRT as well?

If so I will move NativeRuntimeEventSource.PortableThreadPool.Mono.cs in shared and name it to NativeRuntimeEventSource.PortableThreadPool.cs and keep

`<Compile Include="$(MSBuildThisFileDirectory)System\Threading\NativeRuntimeEventSource.PortableThreadPool.cs" Condition="'$(FeatureCoreCLR)' != 'true'" />`

I will add a comment into that file that is currently being shared between Mono and CoreRT to make it more clear and not used on CoreClr.

Reply:
Yes, this works. Thanks!